src/adapter/src/catalog/builtin_table_updates.rs

```diff
@@ -228,7 +228,7 @@ impl CatalogState {
                     row: Row::pack_slice(&[
                         Datum::String(&id.to_string()),
                         Datum::String(column_name.as_str()),
-                        Datum::Int64(i as i64 + 1),
+                        Datum::UInt64(u64::try_from(i + 1).expect("invalid column number")),
```

Comment:
Is this a usize? If so use ore’s cast_from instead. One less expect!

Reply:
Done